flink-core/pom.xml

```diff
@@ -63,6 +63,19 @@ under the License.
 			<artifactId>guava</artifactId>
 			<version>${guava.version}</version>
 		</dependency>
+
+        <dependency>
+            <groupId>com.clearspring.analytics</groupId>
+            <artifactId>stream</artifactId>
+            <version>2.7.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>it.unimi.dsi</groupId>
+                    <artifactId>fastutil</artifactId>
```

Comment:
The library we are using from clearspring implements a series of streaming algorithms. We are only using HyperLogLog and CountMinSketch to estimate count distinct and heavy hitters respectively. As far as I could see fastutil is used only in the implementation of a separate streaming algorithm to track histograms (QDigest). Since we are not using this particular algorithm, it should be ok to exclude fastutil from the dependency.